packages/ui/src/theme/tokens/components/toggleButton.ts

```diff
@@ -7,84 +7,85 @@ import {
   backgroundColor: DesignToken<BackgroundColorValue>;
-}
-interface ToggleButtonActiveTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
-}
-interface ToggleButtonFocusTokens {
   borderColor: DesignToken<BorderColorValue>;
   color: DesignToken<ColorValue>;
 }
+interface ToggleButtonHoverTokens
+  extends Pick<ToggleButtonStateTokens, 'backgroundColor'> {}
+interface ToggleButtonActiveTokens
+  extends Pick<ToggleButtonStateTokens, 'backgroundColor'> {}
+interface ToggleButtonFocusTokens
+  extends Pick<ToggleButtonStateTokens, 'borderColor' | 'color'> {}
 
-interface ToggleButtonDisabledTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
-  borderColor: DesignToken<BorderColorValue>;
-  color: DesignToken<ColorValue>;
-}
+interface ToggleButtonDisabledTokens extends ToggleButtonStateTokens {}
 
-interface ToggleButtonPressedTokens {
-  borderColor: DesignToken<BorderColorValue>;
-  color: DesignToken<ColorValue>;
-  backgroundColor: DesignToken<BackgroundColorValue>;
+interface ToggleButtonPressedTokens
+  extends ToggleButtonPressedStateTokens,
+    ToggleButtonStateTokens {
   _hover: ToggleButtonPressedHoverTokens;
 }
 
-interface ToggleButtonPressedHoverTokens {
+interface ToggleButtonPressedStateTokens {
   backgroundColor: DesignToken<BackgroundColorValue>;
 }
 
-interface ToggleButtonPrimaryTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
+interface ToggleButtonPressedHoverTokens
+  extends ToggleButtonPressedStateTokens {}
+
+interface ToggleButtonPrimaryTokens extends ToggleButtonPrimaryStateTokens {
   borderWidth: DesignToken<BorderWidthValue>;
   _focus: ToggleButtonPrimaryFocusTokens;
   _hover: ToggleButtonPrimaryHoverTokens;
   _disabled: ToggleButtonPrimaryDisabledTokens;
   _pressed: ToggleButtonPrimaryPressedTokens;
 }
-interface ToggleButtonPrimaryFocusTokens {
+
+interface ToggleButtonPrimaryStateTokens {
   borderColor: DesignToken<BorderColorValue>;
   backgroundColor: DesignToken<BackgroundColorValue>;
   boxShadow: DesignToken<BoxShadowValue>;
   color: DesignToken<ColorValue>;
 }
+interface ToggleButtonPrimaryFocusTokens
+  extends ToggleButtonPrimaryStateTokens {}
 
-interface ToggleButtonPrimaryHoverTokens {
-  backgroundColor: DesignToken<BackgroundColorValue>;
-  color: DesignToken<ColorValue>;
-}
+interface ToggleButtonPrimaryHoverTokens
+  extends Pick<ToggleButtonPrimaryStateTokens, 'backgroundColor' | 'color'> {}
 
-interface ToggleButtonPrimaryDisabledTokens {
-  borderColor: DesignToken<BorderColorValue>;
-  backgroundColor: DesignToken<BackgroundColorValue>;
-  color: DesignToken<ColorValue>;
-}
+interface ToggleButtonPrimaryDisabledTokens
+  extends Pick<
+    ToggleButtonPrimaryStateTokens,
+    'backgroundColor' | 'borderColor' | 'color'
+  > {}
 
-interface ToggleButtonPrimaryDisabledTokens {
-  color: DesignToken<ColorValue>;
-}
+interface ToggleButtonPrimaryDisabledTokens
+  extends Pick<ToggleButtonPrimaryStateTokens, 'color'> {}
```

Comment:
Why do we have 2 `ToggleButtonPrimaryDisabledTokens`? Can we combine them together?